packages/pymedphys_pinnacle/src/pymedphys_pinnacle/exporttool/pinnacle.py

```diff
@@ -0,0 +1,158 @@
+import sys
+import os
+import logging
+
+from .pinn_yaml import pinn_to_dict
+from .pinnacle_plan import PinnaclePlan
+from .pinnacle_image import PinnacleImage
+from .rtstruct import convert_struct
+from .rtplan import convert_plan
+from .rtdose import convert_dose
+from .image import convert_image
+
+
+# This class holds all information relating to the Pinnacle Dataset
+# Can be used to export various elements of the dataset, Images,
+# RTSTRUCT, RTPLAN or RTDOSE.
+class Pinnacle:
```

Comment:
I think this might be better named `PinnacleExport` so as not to clash with other pinnacle tools that will be exported from PyMedPhys.

Reply:
Also, could this object be default return pydicom datasets instead of going straight to disk? That means one doesn't need to go on and off the disk if they want to programmatically use the DICOM files themselves.

In the future I would imagine just a single function `pinnacle_dicom_export` that accepts a path and returns a `DicomCollection` object that contains all of the DICOM objects contained at that pinnacle location.

What do you think?

Certainly doesn't have to be like that for this pull request though. But worth thinking about the API